packages/plugin/src/configs/schema-all.ts

```diff
@@ -2,37 +2,22 @@
  * 🚨 IMPORTANT! Do not manually modify this file. Run: `yarn generate-configs`
  */
 
-import { recommendedConfig } from './recommended';
-
-export const allConfig = {
-  ...recommendedConfig,
+export default {
+  extends: ['plugin:@graphql-eslint/base', 'plugin:@graphql-eslint/schema-recommended'],
   rules: {
-    ...recommendedConfig.rules,
     '@graphql-eslint/alphabetize': [
       'error',
       {
         fields: ['ObjectTypeDefinition', 'InterfaceTypeDefinition', 'InputObjectTypeDefinition'],
         values: ['EnumTypeDefinition'],
-        selections: ['OperationDefinition', 'FragmentDefinition'],
-        variables: ['OperationDefinition'],
         arguments: ['FieldDefinition', 'Field', 'DirectiveDefinition', 'Directive'],
       },
     ],
-    '@graphql-eslint/description-style': 'error',
     '@graphql-eslint/input-name': 'error',
-    '@graphql-eslint/match-document-filename': 'error',
-    '@graphql-eslint/no-deprecated': 'error',
-    '@graphql-eslint/no-duplicate-fields': 'error',
-    '@graphql-eslint/no-hashtag-description': 'error',
     '@graphql-eslint/no-scalar-result-type-on-mutation': 'error',
-    '@graphql-eslint/no-unreachable-types': 'error',
     '@graphql-eslint/no-unused-fields': 'error',
```

Comment:
If the intention is to tailor this set to items that deal with schema only and do not require operations, e.g. for a server environment, consider removing this rule.

Reply:
the purpose of `all` config is to include all available rules for schema or operations, so this rule is included explicitly